index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+ 
+
 </head>
+
 <body>
-    <h1 class="tittle"></h1>
-    
+  <header class="tool_bar">
+
+    <a class="Logo" href="">
+      <h2>My Logo</h2>
+    </a>
+
+
+    <nav>
+      <a href=""> <img class="union" src="images/Union.png" alt=""></a>
+    </nav>
+
+  </header>
+
+  <section class="headline-sec">
+
+    <h1 class="greetings">I’m Maurice <br>
+      Glad to see <br> you!</h1>
+
+
+    <p class="text2">
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+
+
+    <h2 class="connect">LET’S CONNECT</h2>
+
```

Comment:
- [ ] Please `hx` tags suppose to present a heading element LET'S CONNECT is a button that should take the user to the contact form, so it should be in `a` tag or a `button` tag